src/Functions/fitParamPolyToSkeleton.m

```diff
@@ -1,15 +1,22 @@
-%% Fits a parameterized polynomial [0,1] -> R^3 to a skeleton point cloud
-% Params: 
-%              skeleton - 3 x N skeleton pointcloud
-%       OPTIONAL degree - degree of fitted polynomial (DEFAULT: 3)
+function [r3polynomial, avgTperMm] = fitParamPolyToSkeleton(skeleton, degree)
+% Fits a parameterized polynomial [0,1] -> R^3 to a skeleton point cloud
+%
+% USAGE:
+%
+%    [r3polynomial, avgTperMm] = fitParamPolyToSkeleton(skeleton, degree)
 %
-% Andreas Husch
-% Centre Hospitalier de Luxembourg / Luxembourg Centre for Systems
-% Biomedicine, University of Luxembourg
-% 2014  - 2017
-% mail@andreashusch.de, husch.andreas@chl.lu
+% INPUTS: 
+%    skeleton:      3 x N skeleton pointcloud
```

Comment:
@adhusch Could you please review the variable descriptions and provide one for those it's missing?